public/index.html

```diff
@@ -4,40 +4,11 @@
     <meta charset="utf-8" />
```

Comment:
Jeśli chodzi o UI:
- nie wyswietlałabym dscription na całej szerokości ekranu, lepiej byłoby na max połowie ekranu moim zdaniem

Reply:
Przy uruchamianiu są 3 erorry, które trzeba naprawic zeby odpalic apkę, nie wiem jak u Ciebie to wygląda.

Reply:
A co to za błędy? U mnie nie ma żadnych.
Co do UI to taki był mój zamysł na początku ale niektóre opisy są długie i za "wysokie" były jak się wyświetlały na ok 60% width. Dodałam też ciemniejsze tło dla opisów bo niektóre zdjęcia mają dużo białych elementów i się wszystko zlewa. Mogę np wyświetlać niepełny opis na 50 czy 60% długości i bez szarego tła lub zostawić jak jest. Daj znać :)

Reply:
@sabinkie ciemniejsze tło wygląda bardzo fajnie. Możesz zmiejszyć width na te 60% i ustawić max-height plus dodać properties z css, które dodają '...' na końcu stringa jeśli jest za długi. Chodzi mi o coś takiego https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_text-overflow

Wtedy jak będzie za dlugi tekst to user bedzie musial kliknąć na film żeby przeczytac więcej